packages/dev/serializers/src/glTF/2.0/glTFSerializer.ts

```diff
@@ -40,6 +41,11 @@ export interface IExportOptions {
      * Indicates if coordinate system swapping root nodes should be included in export
      */
     includeCoordinateSystemConversionNodes?: boolean;
+
+    /**
+     * Indicates the desired export mimetype. Default is png.
+     */
+    mimetype?: ImageMimeType;

```

Comment:
Capital T since mime type is two words.
```suggestion
    mimeType?: ImageMimeType;
```